packages/sanity/src/core/util/draftUtils.ts

```diff
@@ -210,3 +210,16 @@ export function removeDupes(documents: SanityDocumentLike[]): SanityDocumentLike
     .map((entry) => entry.draft || entry.published)
     .filter(isNonNullable)
 }
+
+/**
+ * @beta
+ * Given a document id, it indicates whether it is a draft, version or published document.
```

Comment:
Another super minor nitpick the `@beta` and etc usually come right at the end!

```suggestion
 * Given a document id, it indicates whether it is a draft, version or published document.
 * @beta
```